ground/gcs/src/app/main.cpp

```diff
@@ -263,8 +263,15 @@ int main(int argc, char **argv)
         // fix Mac OS X 10.9 (mavericks) font issue
         // https://bugreports.qt-project.org/browse/QTBUG-32789
         QFont::insertSubstitution(".Lucida Grande UI", "Lucida Grande");
+
+        QSettings tmpSettings;
+
+        bool value = true;
+
+        tmpSettings.setValue(QString("NSAppSleepDisabled"), value);
```

Comment:
Why use 3 different lines to do this? Wouldn't:
---
QSettings().setValue(QString("NSAppSleepDisabled"), true);
accomplish the same thing?